src/update-manager.js

```diff
@@ -100,10 +100,11 @@ class UpdateManager {
     var definitive = false
     var requests = kb.each(undefined, this.ns.link('requestedURI'), docpart(uri))
 
-    // Hack for the moment @@@@ 2016-02-12
+    /* "Hack for the moment" @@@@ 2016-02-12 - Dimitri added (why?); timbl removed 2019-08
```

Comment:
@dmitrizagidulin could we perhaps remove this hack altogether?

Reply:
Always nice to remove code if we can ^_^

Reply:
I'm sorry, I don't remember what the context for that was :) Feel free to remove.

Reply:
Thank you, I'm removing this in the latest commit ^_^